packages/ods/src/components/button/src/components/ods-button/ods-button.tsx

```diff
@@ -22,6 +23,7 @@ export class OdsButton {
   @Prop({ reflect: true }) public color: OdsButtonColor = ODS_BUTTON_COLOR.primary;
   /** @docType OdsIconName */
   @Prop({ reflect: true }) public icon?: OdsIconName;
+  @Prop({ reflect: true }) public iconAlignment?: OdsButtonIconAlignment = ODS_BUTTON_ICON_ALIGNMENT.left;
```

Comment:
though it will be removed with the doc refactor, you need to add an explicit `/** @docType OdsButtonIconAlignment */` here